runtime/trappist/src/lib.rs

```diff
@@ -25,6 +25,7 @@ pub mod constants;
 mod contracts;
 pub mod xcm_config;
 
+use common::DAYS;
```

Comment:
Minor. For consistency i would import `DAYS` from here:
https://github.com/paritytech/trappist/pull/159/files#diff-1f7754547bbff248c6d0c46b32d579f4ecd082d31b014b6d141d116d6f2e0df0R64